pure-pulsars/src/cmds/help_bot.py

```diff
@@ -0,0 +1,20 @@
+"""Help command for the bot."""
+
+import discord
+from discord import app_commands
+
+
+def main(tree: app_commands.CommandTree) -> None:
+    """Create a help command."""
+
+    @tree.command(
+        name="help",
+        description="Responds with help",
+    )
+    async def help_me(interaction: discord.Interaction) -> None:
+        """Display help text from commands available in the bot comamnd tree."""
```

Comment:
Minor spelling mistake. I think the help command could do with a bit more information, such as the parameters that you need to give each command.